payloads/bareflank/Makefile

```diff
@@ -0,0 +1,144 @@
+##
+## This file is part of the coreinfo project.
+##
+## Copyright (C) 2008 Advanced Micro Devices, Inc.
+## Copyright (C) 2008 Uwe Hermann <uwe@hermann-uwe.de>
+##
+## This program is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; version 2 of the License.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+
+src := $(CURDIR)
+srctree := $(src)
+srck := $(src)/../../util/kconfig
+coreinfo_obj := $(src)/build
+objk := $(src)/build/util/kconfig
+
+ifeq ($(filter %clean,$(MAKECMDGOALS)),)
+export KERNELVERSION      := 0.1.0
+export KCONFIG_AUTOHEADER := $(coreinfo_obj)/config.h
```

Comment:
@krystian-hebel this coreinfo_obj stuff looks weird. Does it have any relation to coreinfo payload?

Reply:
@miczyg1 I've copied this MAkefile from coreinfo, there may be more leftovers, I guess I should clear it

Reply:
Please do, This looks confusing